doc/reference/reference_lua/box_stat/memtx.rst

```diff
@@ -0,0 +1,175 @@
+    for trackers per a single transaction.
+  * ``conflicts`` is the memory allocated for *conflicts*
+    which are entities created when transactional conflicts occur.
+    Like in the :ref:`previous sections <box_introspection-box_stat_memtx_tx_total_avg_max>`,
+    Tarantool reports the total, average, and maximal number of allocated bytes.
+  * ``tuples`` is the memory allocated for storing tuples.
+    With MVCC, tuples are stored using the *stories* mechanism. Nearly every
+    tuple has its story. Even tuples in an index may have their stories, so
+    it may be useful to differentiate memory allocated for tuples and memory
+    allocated for stories.
+
+    All stored tuples fall into three categories, with memory statistics
+    reported for each category:
+
+    * ``tracking`` is for tuples that are not used by any transactions directly,
+      but MVCC uses them for tracking transaction reads.
+    * ``used`` is for tuples that are used by active read-write transactions.
+      See a detailed :ref:`example <box_introspection-box_stat_memtx_tx_example>` below.
+    * ``read_view`` is for tuples that are not used by active read-write transactions,
+      but are used by read-only transactions.
+
+      For each of the three categories, Tarantool reports two statistical blocks:
+
+      * ``stories`` is for stories.
+      * ``retained`` is for *retained* tuples which do not belong to any index,
+        but MVCC doesn't allow to delete them yet.
+
+      For each block, Tarantool reports the following statistics:
+
+      * ``count`` is the number of stories or retained tuples.
+      * ``total`` is the number of bytes allocated for stories or retained tuples.
+
+.. _box_introspection-box_stat_memtx_tx_example:
+
+**Example**
+
+This example illustrates memory statistics for ``used`` tuples in a transaction.
+
+The cluster must be started with the :ref:`database.use_mvcc_engine <configuration_reference_database_use_mvcc_engine>`
+parameter set to true. This :ref:`enables MVCC <txn_mode_mvcc-enabling>` so that
+``box.stat.memtx.tx().mvcc`` contained non-zero values.
+
+The next step is to create a space with a primary index and to begin a transaction:
+
+.. code-block:: lua
+
+   box.schema.space.create('test')
+   box.space.test:create_index('pk')
+
+   box.begin()
+   box.space.test:replace{0, 0}
+   box.space.test:replace{0, string.rep('a', 100)}
+   box.space.test:replace{0, 1}
+   box.space.test:replace{1, 1}
+   box.space.test:replace{2, 1}
+
+In the transaction above, three tuples are replaced by the `0` key:
+
+* ``{0, 0}``
+* ``{0, 'aa...aa'}``
+* ``{0, 1}``
+
+MVCC considers all these tuples as ``used`` since they belong to the current transaction.
+Also, MVCC considers tuples ``{0, 0}`` and ``{0, 'aa..aa'}`` as ``retained`` because
+they don't belong to any index (unlike ``{0, 1}``), but they cannot be deleted yet.
+
+Calling ``box.stat.memtx.tx()`` now will bring something like this:
```

Comment:
```suggestion
Calling ``box.stat.memtx.tx()`` now returns the following result:
```

Up to you. As I know, "something like this" is often considered weak or vague writing in tech docs.